prep.sh

```diff
@@ -25,7 +25,7 @@ cp -rv test-classes/test-dependencies/*.hpi megawar/WEB-INF/plugins
 # TODO find a way to encode this in some POM so that it can be managed by Dependabot
 version=0.1.0
 pct=$HOME/.m2/repository/org/jenkins-ci/tests/plugins-compat-tester-cli/$version/plugins-compat-tester-cli-$version.jar
-[ -f $pct ] || $MVN dependency:get -Dartifact=org.jenkins-ci.tests:plugins-compat-tester-cli:$version:jar -Dtransitive=false
+[ -f $pct ] || $MVN dependency:get -Dartifact=org.jenkins-ci.tests:plugins-compat-tester-cli:$version:jar -DremoteRepositories=https://repo.jenkins-ci.org/public/ -Dtransitive=false
```

Comment:
Not sure if this part interacts nicely with the Azure mirror, but not a big deal one way or another.